### ISSUE TITLE
black border in video player

### ISSUE THREAD

**Issue by Kadoffe** [getgrav/grav-plugin-youtube#22]
Hey guys,

I wanted to ask what the reason for the `padding-top: 25px;` in the youtube.css is?
It only causes a black border (top and bottom) inside the video player for me.
With padding-top set to 0 the embedding works just fine and I don't get those borders.

**Comment by rhukster** [comment]
It really depends on the aspect ratio: 

* https://stackoverflow.com/questions/38270661/displaying-a-youtube-video-with-iframe-full-width-of-page
* https://coolestguidesontheplanet.com/videodrome/youtube/

You can override that value and tweak it but it `padding: 25px` is considered the most robust value.

**Comment by angusmcb** [comment]
I realise this issue has been closed a long time, but I still think this is an issue. Whilst it does depend on the aspect ratio of the video, for 16:9 this is 56.25%. For a different aspect ratio it would be a different percentage, but i can't see why it would ever need an additional fixed value in px.

- the first link does have 25px padding top in the answer, but somebody has commented asking why it's there - it's not at all obvious it's required. 
- the second link does not mention the 25px padding top

It's also not on https://css-tricks.com/fluid-width-video/

I created a pull request for this #34
